openebs/openebs-provisioner.go

```diff
@@ -90,7 +93,12 @@ func (p *openEBSProvisioner) Provision(options controller.VolumeOptions) (*v1.Pe
 	volSize := options.PVC.Spec.Resources.Requests[v1.ResourceName(v1.ResourceStorage)]
 	volumeSpec.Metadata.Labels.Storage = volSize.String()
 
-	volumeSpec.Metadata.Labels.StorageClass = *options.PVC.Spec.StorageClassName
+	className := GetPersistentVolumeClass(options)
+
+	if *className == "" {
```

Comment:
Will nil check be better?

Reply:
className is pointer

Reply:
This would imply that classname is never NIL and at least, always has a value namely `""` right? IOW, we always dereference the pointer

Reply:
Yes, changed to pointer `nil` check